jinahub/encoders/image/ImageTorchEncoder/README.md

```diff
@@ -2,24 +2,57 @@
 
 **ImageTorchEncoder** wraps the models from [torchvision](https://pytorch.org/vision/stable/index.html).
 
-**ImageTorchEncoder** encodes `Document` blobs of type a `ndarray` and shape Batch x Height x Width x Channel 
-into a `ndarray` of Batch x Dim and stores them in the `embedding` attribute of the `Document`.
+**ImageTorchEncoder** encodes `Document` blobs of type `ndarray` and shape Height x Width x Channel 
+into an `ndarray` of shape `embedding_dim` and stores them in the `embedding` attribute of the `Document`.
```

Comment:
Can we make this explanation a bit nicer? It should be stated that this blob is an image, for example

Reply:
done